adam-core/src/main/scala/org/bdgenomics/adam/rdd/contig/NucleotideContigFragmentRDD.scala

```diff
@@ -169,7 +170,7 @@ case class NucleotideContigFragmentRDD(
    * @param region Reference region over which to get sequence.
    * @return String of bases corresponding to reference sequence.
    */
-  def getReferenceString(region: ReferenceRegion): String = {
+  def extract(region: ReferenceRegion): String = {
```

Comment:
So you would like ReferenceFile to change its interface to be slice instead of extract?